README.Rmd

```diff
@@ -54,6 +54,18 @@ To execute Rust code, you will also need to set up a working Rust toolchain. See
 
 ## Usage
 
+The status of Rust toolchain and available targets can be checked with `rust_sitrep()`:
+
+```{r eval=FALSE, echo=TRUE}
+rust_sitrep()
+# Rust infrastructure sitrep:
+# ✔ "rustup": 1.26.0 (5af9b9484 2023-04-05)
+# ✔ "cargo": 1.72.0 (103a7ff2e 2023-08-15)
+# ℹ host: x86_64-pc-windows-msvc
+# ℹ toolchain: stable-x86_64-pc-windows-msvc (default)
+# ℹ target: x86_64-pc-windows-gnu
+```
```

Comment:
perhaps, if you know how to repro, can we provide what the output of a _failed_ sitrep would look like so users know how to continue from there?